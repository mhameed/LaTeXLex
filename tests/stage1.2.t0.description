author: mmh
date: 09/02/28
description: 
Making sure that comments are removed outside verbatim, but not inside.
