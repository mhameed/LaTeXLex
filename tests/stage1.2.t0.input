hi this is some text.
Here comes a %comment.
%This line is a comment

\begin{verbatim}
%this is a comment.
\end{verbatim}
