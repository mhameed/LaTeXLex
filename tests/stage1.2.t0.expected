hi this is some text.
Here comes a 


\begin{verbatim}
%this is a comment.
\end{verbatim}
